updatecfg.go

```diff
@@ -168,6 +169,10 @@ func (cmd *updateConfigCommand) run() error {
 		return err
 	}
 
+	if reflect.ValueOf(ls).IsNil() {
```

Comment:
Just curious whether it returned the empty list of `ls.Releases` or `ls` is nil?

Reply:
Please removed commented code if not needed any more, thanks!

Reply:
> Just curious whether it returned the empty list of `ls.Releases` or `ls` is nil?

ls is nil, so ls.Releases will panic.

Reply:
> Please removed commented code if not needed any more, thanks!

Done